app/api/ada/index.js

```diff
@@ -1135,6 +1135,87 @@ export default class AdaApi {
     }
   }
 
+  /**
+   * Restore all addresses like restoreWallet() but do not touch storage.
+   * TBD: this function is based on restoreWallet() and duplicate parts of it.
+   */
+  async restoreWalletForTransfer(
+    recoveryPhrase: string,
```

Comment:
You need to use one `Request` and one `Return` type pattern we use for the rest of this file otherwise it will cause issues when you try and add this to a store.

Reply:
Already realized it when working on the store part. 😂